app/assets/scss/.service-status-removed

```diff
@@ -0,0 +1,7 @@
+.service-status-removed {
+  color: $red;
+}
+
+.service-status-expired {
```

Comment:
Is this used anywhere now?  Or is expired gone from everywhere else in the frontend?